WooCommerce/Classes/ViewRelated/Orders/Order Creation/NotesSection/NewOrderCustomerNotesViewModel.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import Combine
+
+final class OrderNotesViewModel: OrderCreationNotesViewModel {
+    init(originalNote: String = "", analytics: Analytics = ServiceLocator.analytics) {
+        super.init(originalNote: originalNote) {
+
+        }
+    }
+
+
+}
```

Comment:
I'm not sure if I understand the purpose of this subclass.

I'm thinking that we can rename `OrderCreationNotesViewModel ` to something like `LocalOrderNoteViewModel` and reuse it in most places.

or am I missing something? 😅

Reply:
I completely removed this as we're leaving the Simple Payments notes refactoring to another PR and changing how to trigger the Customer note analytics call, so now I defined a specific ViewModel for the New Order notes editing, it introduces some code repetition but it can be fixed in another PR as you suggested!